### DISCUSSION TITLE
Edit files failed: http error 500

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#493] [Report a bug] [answered]
I've just updated the Yellow and now can't edit, save or create new files in web-interface.
After I press Save button I've got the http error 500.

Same situation in different browsers.

Datenstrom Yellow 0.8.10 = for current website version
= for current error message
2020-03-07 17:58:03 info Update extension 'Edit 0.8.22'
2020-03-07 17:58:03 info Update extension 'English 0.8.19'
2020-03-07 17:58:03 info Update extension 'Feed 0.8.7'
2020-03-07 17:58:03 info Update extension 'German 0.8.19'
2020-03-07 17:58:03 info Update extension 'Image 0.8.7'
2020-03-07 17:58:03 info Update extension 'Markdown 0.8.12'
2020-03-07 17:58:03 info Update extension 'Russian 0.8.19'
2020-03-07 17:58:03 info Update extension 'Update 0.8.17'

I see new commits in core with no version increment — so I think it's not good in this case.

> Description: Core functionality of the website.
> Published: 2020-03-05 21:44:50	Published: 2020-03-07 18:04:06

How can I know witch one is good: 0.8.10 or 0.8.10? :)
@markseu, please check it.

**Comment by schulle4u** [comment, marked as answer]
I can confirm the problem. 
```
PHP Fatal error:  Uncaught Error: Call to undefined method YellowToolbox::normaliseLines() in /home/users/[...]/system/extensions/edit.php:1064 [...]
```

The installed core extension is not compatible with latest edit extension. Please [manually download and re-install the core extension](https://github.com/datenstrom/yellow-extensions/tree/master/features/core), this will solve the problem. 

Steffen

**Comment by markseuffert** [comment]
I am really sorry about that, I missed it when uploading new extensions two days ago. 

Please do as Steffen described and Core 0.8.11 is now available. Does this fix the problem?

**Comment by i-do** [comment]
Manual download and install the core extension fixed the issue.
@schulle4u, @markseu — thanks for fast reaction!
